docs/src/components/sender-compat.md

```
@@ -0,0 +1,576 @@
+```
+
+:::tip 数据结构说明
+SenderCompat 保持 v0.3.0 的数据结构：
+```typescript
+// ✅ SenderCompat 使用
+{ type: 'template', content: '...' }
+
+// ⚠️ v0.4.0 Sender 使用（注意类型名变更）
+{ type: 'block', content: '...' }
+```
+详见 [模板填充迁移](#模板迁移)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)